pom.xml

```diff
@@ -577,7 +577,7 @@
       <dependency>
         <groupId>${project.groupId}</groupId>
         <artifactId>netty-tcnative</artifactId>
-        <version>1.1.32.Fork1</version>
+        <version>1.1.32.Fork2-SNAPSHOT</version>
```

Comment:
How do we typically coordinate the version changes?  Followup PR when the next version of netty-tcnative is released, or release netty-tcnative before pulling this in and update the version?